plugins/ltac2/g_ltac2.mlg

```diff
@@ -196,8 +202,10 @@ GRAMMAR EXTEND Gram
         { CAst.make ~loc @@ CTacCst (AbsKn (Tuple 0)) }
       | "["; a = LIST0 ltac2_expr LEVEL "5" SEP ";"; "]" ->
         { Tac2quote.of_list ~loc (fun x -> x) a }
+      | "{"; test_ident_with_or_lpar_or_rbrac; e = ltac2_expr LEVEL "0"; "with"; a = tac2rec_fieldexprs; "}" ->
```

Comment:
I'd rather have the OCaml syntax with parentheses instead of a record itself...

Reply:
I don't understand.

Reply:
I think @ppedrot is suggesting
```suggestion
      | "("; test_ident_with_or_lpar_or_rbrac; e = ltac2_expr LEVEL "0"; "with"; a = tac2rec_fieldexprs; ")" ->
```
(except maybe `test_ident_with_or_lpar_or_rbrac` needs to change?)

Reply:
I don't understand why, though

Reply:
Ah, no sorry, I misread the parsing rule. I think this is OK as it is currently.